embrace-android-core/src/main/kotlin/io/embrace/android/embracesdk/internal/config/behavior/OtelBehaviorImpl.kt

```diff
@@ -0,0 +1,18 @@
+package io.embrace.android.embracesdk.internal.config.behavior
+
+import io.embrace.android.embracesdk.internal.config.UnimplementedConfig
+import io.embrace.android.embracesdk.internal.config.remote.RemoteConfig
+
+/**
+ * Provides the behavior for OpenTelemetry configuration
+ */
+class OtelBehaviorImpl(
+    override val remote: RemoteConfig?,
+) : OtelBehavior {
+
+    override val local: UnimplementedConfig = null
+
+    override fun shouldUseKotlinSdk(): Boolean {
+        return remote?.killSwitchConfig?.disableOtelKotlinSdk != true
```

Comment:
Won't this default the return value to `true` if `remote?.killSwitchConfig?.disableOtelKotlinSdk` is null? IMO we need to disable this behavior by default and have a % rollout due to the risk of the change.